### PR TITLE
DBZ-3500 Incremental snapshot follow-up tasks

### PR DESCRIPTION
**Pull request by jpechane**

### CODE REVIEW

File: debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java [hunk 1]
```diff
@@ -1458,4 +1458,22 @@ public PreparedStatement readTablePreparedStatement(CommonConnectorConfig connec
...
+        for (int i = 0; i < columnArray.getColumns().length; i++) {
+            row[columnArray.getColumns()[i].position() - 1] = getColumnValue(rs, i + 1,
+                    columnArray.getColumns()[i], table, databaseSchema);
+        }
+        return row;
+    }
+
+    public String quotedTableIdString(TableId tableId) {
+        return tableId.toDoubleQuotedString();
+    }
```
Comment (gunnarmorling) [inline debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java:1478]: Worth a JavaDoc comment given it's sort of an SPI.

**Review by gunnarmorling — COMMENTED** [review]
One small comment, nothing much to say otherwise :)

File: debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java (hunk 1 above)
Comment (jpechane) [reply debezium-core/src/main/java/io/debezium/jdbc/JdbcConnection.java:1485]: @gunnarmorling Added

**Review by gunnarmorling — APPROVED** [review]

**Comment by leonardBang** [comment]
Thanks @jpechane for the great work for DDD-3 !!! It's really nice work. 
One minor question that we missed `DELETE` event test in `AbstractIncrementalSnapshotTest`. Could I open an issue and add it?

**Comment by jpechane** [comment]
@leonardBang Thanks for the kind words. Yes, please any improvements in test coverage are definely welcome!

**Pull request merged**